index.html

```diff
@@ -31,4 +31,4 @@
       Read more
     </a>
   </body>
-</html>
+</html>
```

Comment:
Please, try not to have newlines at the end of your file. It makes your code larger than required.